### PR TITLE
chore: cleanup history

### LINKED ISSUE #7627: chore: HISTORY.md is missing various PRs
I have not yet isolated the reason, but HISTORY.md is still missing PRs (e.g. #6884). The corresponding PRs do not appear to have the release report from keyman-server either.

We should revisit this together with ~~#7351~~ #7278.

### PR DESCRIPTION
**Pull request by mcdurdin**
Cleans up history -- some entries missing due to way they were merged into master.

Fixes #7627.

@keymanapp-test-bot skip

### CODE REVIEW

File: resources/build/version/src/fixupHistory.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ export const fixupHistory = async (
   let pulls: PRInformation[] = [];
 
   try {
-    pulls = await reportHistory(octokit, base, force, true);
+    pulls = await reportHistory(octokit, base, force, false);
```
Comment (darcywong00) [inline resources/build/version/src/fixupHistory.ts:196]: Is this change intentional?

File: HISTORY.md [hunk 2]
```diff
@@ -1569,14 +1573,14 @@
 
 ## 15.0.101 alpha 2021-08-23
 
-* refactor(web): overhaul of OSK key layout calcs + styling, merges desktop & touch logic 📐 (#5462)
+* refactor(web): overhaul of OSK key layout calcs + styling, merges desktop & touch logic (#5462)
```
Comment (darcywong00) [inline HISTORY.md:1576]: Do we want to also scrub `:cherries:` from HISTORY.md?

File: resources/build/version/src/fixupHistory.ts (hunk 1 above)
Comment (mcdurdin) [reply resources/build/version/src/fixupHistory.ts:196]: Yes, addresses https://github.com/keymanapp/keyman/issues/7627#issuecomment-1321264070. We will watch it and see what happens with 16.0-stable cherry-picks.

**Review by darcywong00 — APPROVED** [review]
lgtm
